### ISSUE TITLE
Anvato Video Embed won't fullscreen on Android

### ISSUE THREAD

**Issue by GeminiLegend** [ionic-team/ionic#13227]
**Ionic version:** 
[ ] **2.x**
[x ] **3.x**
[ ] **4.x**

**I'm submitting a ...**  (check one with "x")
[x ] bug report
[ ] feature request

**Current behavior:**
On Android opening an article in our app with an Anvato video embed (iframe) and playing the video works fine. Attempting to go fullscreen does nothing, it doesn't open the native fullscreen player you usually get when watching an kind of video fullscreen. iOS works flawlessly

**Expected behavior:**
Anvato Embedded video's can play in fullscreen on Android

**Steps to reproduce:**
Articles in our app are iframed in from our WP site. Specifically they are AMP articles. You would need to iframe a WP page with an Anvato embed into an ionic app. Please spare me i didn't choose this method.

Here is repository with a test case of an Anvato embed: `https://github.com/GeminiLegend/AnvatoTestCase`
**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
Cordova CLI: 7.0.1 
Ionic Framework Version: 3.0.1
Ionic CLI Version: 2.2.3
Ionic App Lib Version: 2.2.1
Ionic App Scripts Version: 1.3.7
ios-deploy version: 1.9.1 
ios-sim version: Not installed
OS: macOS Sierra
Node Version: v6.9.1
Xcode version: Xcode 9.0.1 Build version 9A1004
```
This is a show stopping bug with our apps so help would be greatly appreciated.
I've already attempted to use this this fix but to no avail since it doesn't seem to be supported anymore. ```https://stackoverflow.com/questions/36301976/video-doesnt-go-fullscreen-in-ionic```

**Comment by kensodemann** [comment]
Hello! Thank you for opening an issue with us! Would you be able to provide a sample application via GitHub that demonstrates the issue you are having? Thanks for using Ionic!

**Comment by GeminiLegend** [comment]
I will try to make a bare bones app on github that can pull articles from our stream but that might take a bit, the app is quite complicated. Until then if you know of any possible quick fixes that might resolve this i'd be very grateful for the help

**Comment by kensodemann** [comment]
I don't know any, but you can ask this question on our [forum](https://forum.ionicframework.com) or on our [slack channel](https://ionicworldwide.herokuapp.com/). That will reach a much wider audience, some of whom may have dealt with this issue already.

If you do that and find something out, please be sure to copy that info back here.

**Comment by GeminiLegend** [comment]
@kensodemann I've added a test case with an anvato embed

**Comment by GeminiLegend** [comment]
@kensodemann I've updated the app to the latest Ionic packages and the error still persists

**Comment by kensodemann** [comment]
@GeminiLegend - thank you for the sample application. I can confirm that this is is not an issue with Ionic. I took the markup for your iframe and added it to a simple Vue.js web application that I have laying around for educational purposes and deployed it. Running on iOS, it works fine. Running on Android, it does not go full screen.

Since this was not an Ionic application at all, or even an angular application for that matter (actually, it is barely a Vue.js application, more or less just a web-page at the point I tested it, the problem is not connected in any way to the Ionic Framework.
